test/factory/common_factory_tests.dart

```diff
@@ -1,12 +1,12 @@
 import 'dart:js';
 
+import 'package:js/js_util.dart';
```

Comment:
so package:js/js_util.dart actually just exports `dart:js_util` so id say using it is better than depending on an external library. but you could also just keep using js_interop_helpers.dart until its fully removed too. your call.
```suggestion
import 'dart:js_util';
```